_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,55 @@
+
+<!--more-->
+
+First of all, why create a Snap Mini? Here are the main advantages of having your own app inside [Snapchat](https://www.snapchat.com/):
+
+## Easy to Develop
+Snap Minis exist inside the main Snapchat application. They are built on HTML5 and take advantage of the existing Snapchat app, which means they work cross-platform and are readily available to all Snapchat users, reducing the scope of the project.
+
+Additionally, Minis work on both Android and iOS devices. You do not need two different applications to cater to different users.
+
+## Easy to Use
+The Minis are lightweight applications easily accessible through the Chat interface of the Snapchat app. Users do not need to install a separate app and switch applications to use your service. They also don't need to wait for a download to finish.
+
+Snapchat users can launch a Snap Mini from within a conversation they're having with their friends. You can quickly collaborate or play with your friends, without having to open a new app or website.
+
+## Easy to Share with Friends
+A Mini can interact with both the chat and camera interfaces of the Snapchat application. This makes it easy for your users to share that they've used your mini with their friends.
+
+Their friends can then tap the chat card or camera sticker and be taken to your mini. All without leaving Snapchat.
+
+# What Can I do with Minis?
+
+There are several possibilities here. Minis can integrate with the Snapchat features and perform a variety of actions.
+
+Having in mind the concept of a Mini, a bite-sized application inside the Chat interface, you can define an actionable use case within your product and turn it into a mini. Some examples shared by Snap during the Snap Partner Summit 2020 include deciding what to watch, buying tickets, voter registration and health and fitness. If you have a specific action that can be performed by your audience, you can turn it into a Mini.
+
+Another advantage is adding a social aspect to the experience. These mini applications live inside the Chat interface and allow friends to browse and use them together. You can take advantage of the communication that's already happening on Snapchat and bring that into your service.
+
+A pretty awesome feature of minis is you can see in real time which of your friends are online and on that same mini at the same time as you.
+
+If you have an actionable use case that includes a strong social aspect, Minis are perfect for you.
+
+To know more about Minis and hear the experience of some of the early partners working with Snap on this, you can check out this video shared by the company.
+
+[![Snap Partner Summit 2020](http://img.youtube.com/vi/ZoKlrgyKH0Y/0.jpg)](https://www.youtube.com/watch?v=ZoKlrgyKH0Y "Snap Partner Summit 2020")
+
+# How We Can Help
+
+We have developed a Snap Mini using [Vue.js](https://vuejs.org) and [shared our experience doing so here](link-to-ariels-article).
```

Comment:
Maybe we want to say that we are also developing other mini using ReactJS, it can help getting more people interested if they see we can use different technologies (React is more popular than Vue)

Reply:
The idea here is to link to your article when it's ready. I don't like the idea of disclosing we are currently working on another mini. With the one we already worked on, there is no clear implication that we worked with Snap. We could have worked with any other of their partners. Adding that we are currently working on a mini, specially considering how Snap played this out, is a different story. My main concerns about this article really are on what we can and cannot disclose. And right now we can't really disclose too much unfortunately.